hearts/static/room_ui.js

```diff
@@ -67,7 +67,14 @@ function RoomUI(callbacks) {
       buttonHtml = ('<button id="pass_button">' +
         'Pass 3 cards ' + direction + '</button>');
       $('#status').html(buttonHtml);
-      $('#pass_button').click(this.passButtonClickCallback);
+      var that = this;
+      $('#pass_button').click(function(event) {
+        that.passButtonClickCallback();
+      });
+    }
+    else {
```

Comment:
style: `else` should be on the same line as preceding `}`

Reply:
This else statement should be unnecessary once you've added the `hideActionButtons` method, right?

If it is needed, I'd prefer having a single function that hides the pass button (`hideActionButtons` is probably fine for this), and calling it from here instead of duplicating the logic.

Reply:
Fixed.